### DISCUSSION TITLE
22 October 2024 2200 UTC

### DISCUSSION THREAD

**Discussion by ktoddbrown** [ktoddbrown/SoilDRaH#56] [RegularCalls]
### Agenda

These calls are lead by @ktoddbrown and are generally open calls.
Please contact them for call in information.
The purpose of these meetings are to move forward current issues related to the development of Soil-DRaH.
This may include teaching, mutual accountability, code review, paired programming, or other activities.

## (5 | 2200) Check ins
- Who is here
- Note taking?
- How are you showing up? <https://feelingswheel.com/>

## (20 | 2205) Issues checkins
- *What are you working on and what is your next step?*

## (20 | 2225) Data integration walk through
- *KTB live coding how the data integration for ISCN4 is shaping up.*

## (5 | 2245) Backlog
- *What new issues need to be created?*

**Comment by GabrielThrasher** [comment]
### (5 | 2200) Check ins
Who is here - Maggie, Graham, Gabriel, Kate,  KTB, Vaasuki
Note taking? - Gabriel
How are you showing up? https://feelingswheel.com/
### (20 | 2205) Issues checkins
What are you working on and what is your next step?

- Maggie - Been going through the papers that KTB sent on data ingest. Going to reach out to people who wrote the R scripts on slack to get more insight: aiming to get a better understanding of why we choose that data.  Pick two data sets from data1 for the collections.

- Graham - Been working on the Longleaf pine annotations; fixed some mismatched file names. Write up a data report for longleaf. 

- Gabriel - Working on creating how-to for data annotations generation. Pangea data annotations; take a look at the Forest inventory analysis and NRCS. Talked about how to manage the Wiki To-do issue.

- Kate - Working with israd data set. Taking the flatten data and transposing it into one data.  Working on the read israd function. Look at the ISCN4 code (https://github.com/ktoddbrown/SoilDRaH/tree/28-iscn4/vignettes/articles)  shows examples of how to work with the type of datasets. Encapsulate code into functions; work with quarto files. Code review in two weeks? Yes!

- KTB - Went through and got the plots working for CPEAT. The report is not documented yet; needs commeting. The read function (check in with brandon).  CPEAT is close to being merged with curate to see where in breaks.

- Vaasuki - Working with a lot of data sets (cpeat/pangea); writing about them in the manuscript. Ran the code line by line to see what was going on. Trying to figure out how to write the manuscripts. https://doi.org/10.1016/j.ecoinf.2021.101374

### (20 | 2225) Data integration walk through
KTB live coding how the data integration for ISCN4 is shaping up.

- Have a source variable (mapping from) and a target variable (mapping to)

- Non-standard base units (like tenth of an inch) to be able to store numbers as integers instead of floats since integers take up less memory (space) on a computer than floats

- Action column where each instance is treated as note-to-self action items

- Review the methods relating to the data set to be able to appropriately map to the variables that we need.

- Todo: modify the CPEAT read package

- CPEAT combines graphical and tabular data -- can be a bit confusing.

- Looked at CPEAT long format

- Next time well code the annotations for this.

### (5 | 2245) Backlog
What new issues need to be created?
### (5 | 2255) Check outs [appreciations, discoveries, and ToDo]
